snippets/csharp/VS_Snippets_CLR/formatting.numeric.custom/cs/literal2.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+public class Example
+{
+   public static void Main()
+   {
+      double n = 123.8;
+      Console.WriteLine($"{n:#,##0.0K}");  
+   }
+}
+// The example displays the following output:
+//      123.8K
```

Comment:
Suggestion: this snippet contains 12 lines. Only 4 of those contribute to the topic.
What about the following organization (with including only `Snippet1` into docs):
```csharp
using System;

public class Example
{
   public static void Main()
   {
      // <Snippet1>
      double n = 123.8;
      Console.WriteLine($"{n:#,##0.0K}");  
      // The example displays the following output:
      //      123.8K
      // </Snippet1>
   }
}
```

Then, readers would see the following in the published docs:
```csharp
double n = 123.8;
Console.WriteLine($"{n:#,##0.0K}");  
// The example displays the following output:
//      123.8K
```

Short and to the point. And if it's `csharp-interfactive`, it's even better.

The same remark is applicable to another snippet file, *literal1.cs*, as well.

In general, if possible, I suggest to avoid using the following structure as it introduces a lot of noise around the code that matters:
```csharp
using System;

public class Example
{
   public static void Main()
   {
      // Example code
   }
}
```

Obvious exceptions: multiclass examples, a lot of additional `using` statements.